packages/webex-node/package.json

```
@@ -0,0 +1,68 @@
+{
+    "name": "webex-node",
```

Should we consider naming this either,
```suggestion
    "name": "webex/node",
```

or

```suggestion
    "name": "@webex/node",
```

Just for uniformity?

---

@mkesavan13 I didn't name it with scoped `@webex` or `webex/` because I wanted this to be exactly the same as the `webex` package. It's just to let developers know that it's `webex` but for Node.js.